gitops/openshift/apps/efiling-reviewer/template/efiling-reviewer.yaml

```diff
@@ -0,0 +1,249 @@
+                memory: 4Gi
+              requests:
+                cpu: '1'
+                memory: 512Mi
+            readinessProbe:
+              httpGet:
+                path: /actuator/health
+                port: 8080
+                scheme: HTTP
+              initialDelaySeconds: 120
+              timeoutSeconds: 10
+              periodSeconds: 10
+              successThreshold: 1
+              failureThreshold: 3
+            terminationMessagePath: /dev/termination-log
+            name: ${COMPONENT_NAME}
+            livenessProbe:
+              httpGet:
+                path: /actuator/health
+                port: 8080
+                scheme: HTTP
+              initialDelaySeconds: 110
+              timeoutSeconds: 10
+              periodSeconds: 10
+              successThreshold: 1
+              failureThreshold: 3
+            env:
+              - name: CLAMAV_HOST
+                value: clamav
+              - name: CLAMAV_PORT
+                value: '3310'
+              - name: CLAMAV_TIMEOUT
+                value: '150000'
+              - name: REDIS_HOST
+                value: redis
+              - name: REDIS_PORT
+                value: '6379'
+              - name: REDIS_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: redis
+                    key: database-password
+              - name: DILIGEN_BASE_PATH
+                valueFrom:
+                  secretKeyRef:
+                    name: diligen-api
+                    key: DILIGEN_BASE_PATH
+              - name: DILIGEN_PROJECT_IDENTIFIER
+                value: '2'
+              - name: SPLUNK_SOURCE
+                value: efiling-reviewer-api
+              - name: spring_profiles_active
+                value: splunk
+              - name: AIR_SPRING_DATA_MONGODB_HOST
+                value: mongodb
+              - name: AIR_SPRING_DATA_MONGODB_DATABASE
+                valueFrom:
+                  secretKeyRef:
+                    name: mongodb
+                    key: database-name
+              - name: AIR_SPRING_DATA_MONGODB_USERNAME
+                valueFrom:
+                  secretKeyRef:
+                    name: mongodb
+                    key: database-user
+              - name: AIR_SPRING_DATA_MONGODB_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: mongodb
+                    key: database-password
+              - name: AIR_SPRING_DATA_MONGODB_AUTHENTICATION_DATABASE
+                valueFrom:
+                  secretKeyRef:
+                    name: mongodb
+                    key: database-name
+              - name: KEYCLOAK_SSL_REQUIRED
+                value: none
+              - name: KEYCLOAK_RESOURCE
+                value: ai-reviewer-api
+              - name: KEYCLOAK_AUTH_SERVER_URL
+                valueFrom:
+                  secretKeyRef:
+                    name: keycloak-efiling-reiewer
```

Comment:
This secret name has a typo; it is missing the "v" in _reviewer_

Reply:
Thanks @gpoleze, this is corrected